### PR TITLE
WIP - jBPM + OptaPlanner Spring Boot "Flight Seating" example

### PR DESCRIPTION
**Pull request by ge0ffrey**
Initial example. Pretty UI will come in a separate PR.

### CODE REVIEW

File: jbpm-optaplanner-springboot-example/pom.xml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  </parent>
+  <groupId>org.kie.kogito.examples</groupId>
+  <artifactId>jbpm-optaplanner-springboot-example</artifactId>
+
+  <name>Kogito Example :: jBPM, OptaPlanner and SpringBoot</name>
+  <description>Wedding process and optimization</description>
+
+  <properties>
+    <springboot.version>2.2.2.RELEASE</springboot.version>
+    <version.org.optaplanner>7.32.0-SNAPSHOT</version.org.optaplanner>
```
Comment (mswiderski) [inline jbpm-optaplanner-springboot-example/pom.xml:17]: we can't use snapshot versions

File: jbpm-optaplanner-springboot-example/pom.xml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-springboot-starter</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>process-management-addon</artifactId>
```
Comment (mswiderski) [inline jbpm-optaplanner-springboot-example/pom.xml]: do you need this dependency?

File: jbpm-optaplanner-springboot-example/pom.xml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>persistence</id>
```
Comment (mswiderski) [inline jbpm-optaplanner-springboot-example/pom.xml]: this is a straight through process so don't think persistence will be ever used, so I'd opt at removing this profile altogether

File: jbpm-optaplanner-springboot-example/src/main/resources/application.properties [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# The solver runs for 5 seconds. To run for 5 minutes use "5m" and for 2 hours use "2h".
+optaplanner.solver.termination.spent-limit=5s
+
+spring.mvc.servlet.path=/docs
+
+resteasy.jaxrs.scan-packages=org.kie.kogito.**
+
+infinispan.remote.server-list=127.0.0.1:11222
```
Comment (mswiderski) [inline jbpm-optaplanner-springboot-example/src/main/resources/application.properties]: if persistence maven profile is removed this property can be removed as well

File: jbpm-optaplanner-springboot-example/pom.xml [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    <!-- test related -->
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <scope>test</scope>
```
Comment (mswiderski) [inline jbpm-optaplanner-springboot-example/pom.xml]: seems not used

File: jbpm-optaplanner-springboot-example/pom.xml (hunk 1 above)
Comment (ge0ffrey) [reply jbpm-optaplanner-springboot-example/pom.xml:17]: I'll put this PR on draft until 7.32 is released. 7.31 doesn't contain the necessary bits.

File: jbpm-optaplanner-springboot-example/pom.xml (hunk 2 above)
Comment (ge0ffrey) [reply jbpm-optaplanner-springboot-example/pom.xml]: No, removing it. thanks

File: jbpm-optaplanner-springboot-example/pom.xml (hunk 5 above)
Comment (ge0ffrey) [reply jbpm-optaplanner-springboot-example/pom.xml]: Less is more :)

File: jbpm-optaplanner-springboot-example/pom.xml (hunk 1 above)
Comment (ge0ffrey) [reply jbpm-optaplanner-springboot-example/pom.xml:17]: I can't put it as draft now, so I just added "WIP" to the title

**Comment by evacchi** [comment]
Has this PR been superseded by #121 ?

**Comment by ge0ffrey** [comment]
Closing in favor of https://github.com/kiegroup/kogito-examples/pulls

**Pull request closed without merging**